src/routes/graph/update-pin-groups.ts

```diff
@@ -0,0 +1,35 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithClient } from '../../comms/controller';
+
+interface Request {
+  body: {
+    hashIds: string[];
+    mapLayer?: {
+      key: string,
+      includeEdges?: boolean,
+    };
+  };
+}
+
+type Response = void;
+
+const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
+  method: 'post',
+  path: '/update-pin-groups/:hashId',
+  body: Joi.object().keys({
+    hashIds: Joi.array().items(Joi.string()).required(),
+    mapLayer: Joi.object().keys({
+      key: Joi.string().invalid('nodes'),
+      includeEdges: Joi.boolean(),
```

Comment:
Sorry I missed this, I think you need a `.default(false|true)` here ? (from what I've seen in other places in the codebase)